### ISSUE TITLE
Show all comments instead of featured comments by default

### ISSUE THREAD

**Issue by ideallical** [coralproject/talk#4108]
## Search Terms

tabs, featured comments, FEATURED_COMMENTS, ALL_COMMENTS

## Suggestion

Currently the featured comments tab is active on the comment stream in case there are featured comments.
However, some of our users are confused, and would like to see all comments by default.
I'm able to disable this by removing these lines:
https://github.com/coralproject/talk/blob/v7.4.3/src/core/client/stream/tabs/Comments/Stream/StreamContainer.tsx#L229-L233

However, it would be nice if there was an ENV for this, so I do not have to change the core code.

## Use Cases
Some people prefer seeing all comments first instead of featured comments.

**Comment by jbotte** [comment]
100% agree with this request, I do not use this feature because I worry it will confuse readers as well.

I think either showing featured comments above the normal comment feed or having the normal comment feed show first and then users can tab over to featured comments is an overall better user experience.

**Comment by antonborgstrom** [comment]
I agree, placing featured comments above the normal comment feed would be most logical behavior as it is standard in other commenting systems and forums. 

Our users are also confused by the behavior now when all normal comments are hidden by default if there are featured comments in the feed.
